packages/fxa-content-server/app/scripts/views/survey.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*eslint-disable no-unused-vars*/
+import React, { useState, useEffect } from 'react';
+/*eslint-enable no-unused-vars*/
+import ReactDOM from 'react-dom';
+import Cocktail from 'cocktail';
+import BaseView from './base';
+import SettingsPanelMixin from './mixins/settings-panel-mixin';
```

Comment:
Are we only showing the surveys within settings?  I didn't get that impression from reading some of the docs.

I also can't tell what from the settings panel mixin is used in here.

Reply:
This was just to get the survey rendering quickly. The method you have presented in https://github.com/mozilla/fxa/compare/master...chenba:1708-survey-targeting works better